kmk/bootcfg.py

```diff
@@ -86,6 +86,11 @@ def bootcfg(
 
         usb_cdc.enable(data=True)
 
+    # sense not provided -> Skip boot configuration that may disable debug or
+    # rescue facilities.
+    if sense is None:
+        return False
+
     # sense pulled low -> Skip boot configuration that may disable debug or
     # rescue facilities.
     if not sense.value:
```

Comment:
I think we can merge these two branches.
```suggestion
    # sense not provided or pulled low -> Skip boot configuration that may disable debug or
    # rescue facilities.
    if sense is None or not sense.value:
        return False
```

Reply:
Wow, what a great visual for the diff, github.